amundsen_application/static/js/interfaces/Dashboard.ts

```diff
@@ -14,8 +14,8 @@ export interface DashboardMetadata {
   group_name: string;
   group_url: string;
   last_run_state: string;
-  last_run_timestamp: number;
-  last_successful_run_timestamp: number;
+  last_run_timestamp: number | null;
```

Comment:
Are there actually instances where this data would be missing? if so why?

Reply:
It seems so, we have been checking for it all over the place.
Why, no idea!

Reply:
that's weird... I would think that we would always get whatever the last good run was

Reply:
Maybe that's the case now, but I haven't dig deeper, I just want to get rid of that typing issue.